rs/tests/boundary_nodes/BUILD.bazel

```diff
@@ -105,6 +105,7 @@ system_test_nns(
         "k8s",
     ],
     target_compatible_with = ["@platforms//os:linux"],  # requires libssh that does not build on Mac OS
+    uses_boundary_guestos = True,
```

Comment:
not strictly necessary since we infer `uses_boundary_guestos` from the `BOUNDARY_NODE_GUESTOS_RUNTIME_DEPS`